operator/internal/lifecycle/testdata/cases.resources.golden.txtar

```diff
@@ -1004,50 +1005,3 @@
             defaultMode: 272
             secretName: basic-test-default-cert
   status: {}
-- apiVersion: v1
-  automountServiceAccountToken: false
-  kind: ServiceAccount
-  metadata:
-    creationTimestamp: null
-    labels:
-      app.kubernetes.io/instance: basic-test
-      app.kubernetes.io/managed-by: Helm
-      app.kubernetes.io/name: console
-      app.kubernetes.io/version: v2.8.0
-      cluster.redpanda.com/namespace: basic-test
-      cluster.redpanda.com/operator: v2
-      cluster.redpanda.com/owner: basic-test
-      helm.sh/chart: console-0.7.31
-      helm.toolkit.fluxcd.io/name: basic-test
-      helm.toolkit.fluxcd.io/namespace: basic-test
-    name: basic-test-console
-    namespace: basic-test
-- apiVersion: v1
-  kind: Service
-  metadata:
-    creationTimestamp: null
-    labels:
-      app.kubernetes.io/instance: basic-test
-      app.kubernetes.io/managed-by: Helm
-      app.kubernetes.io/name: console
-      app.kubernetes.io/version: v2.8.0
-      cluster.redpanda.com/namespace: basic-test
-      cluster.redpanda.com/operator: v2
-      cluster.redpanda.com/owner: basic-test
-      helm.sh/chart: console-0.7.31
-      helm.toolkit.fluxcd.io/name: basic-test
-      helm.toolkit.fluxcd.io/namespace: basic-test
-    name: basic-test-console
-    namespace: basic-test
-  spec:
-    ports:
-    - name: http
-      port: 8080
-      protocol: TCP
-      targetPort: 0
-    selector:
-      app.kubernetes.io/instance: basic-test
-      app.kubernetes.io/name: console
-    type: ClusterIP
-  status:
-    loadBalancer: {}
```

Comment:
These are both Console resources that no longer render due to the chart upgrade.